app/test/unit/create-branch-test.ts

```diff
@@ -29,6 +29,8 @@ const defaultBranch: Branch = {
   upstreamWithoutRemote: null,
   nameWithoutRemote: 'my-default-branch',
   isDesktopForkRemoteBranch: false,
+  ref: '',
```

Comment:
If you agree with my suggestion above:
```suggestion
  ref: null,
```

Reply:
Decided to leave as string and as such want ref to be an empty string on unit tests to not add something that appears meaningful.